README.md

```diff
@@ -24,12 +24,8 @@ To get the project running, follow these steps:
 
 1. Open the backend folder (WebApiBackend) and find the Solution (.sln) file, and open in Visual Studio.
 2. Once Visual Studio loads up, you should have a green "IIS Express" button to start the app in the top toolbar. Click it.
-3. It'll take some time the first run (as Visual Studio should install all dependencies). Not much will appear to happen once the backend server is running.
+3. It'll take some time the first run (as Visual Studio should install all dependencies). Once the backend server is running, a new window will open displaying Swagger.
```

Comment:
Maybe a brief explanation of what Swagger does in this context? (I didn't know until I recently ran master haha!)

Reply:
Updated, please review again :)